components/input.css

```diff
@@ -0,0 +1,54 @@
+.input-wrap {
+  font-family: sans-serif;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.textbox {
+  display: flex;
```

Comment:
Just a suggestion that you could have made utility classes for this as this is most commonly used property .

Reply:
done with changes plz approve